api/api_ethereum.go

```diff
@@ -1571,13 +1577,18 @@ func EthDoCall(ctx context.Context, b Backend, args EthTransactionArgs, blockNrO
 	// this makes sure resources are cleaned up.
 	defer cancel()
 
-	// TODO-Klaytn: Klaytn is using fixed baseFee as now but, if we change this fixed baseFee as dynamic baseFee, we should update this logic too.
-	fixedBaseFee := new(big.Int).SetUint64(params.ZeroBaseFee)
+	// header.BaseFee != nil means kip71 hardforked
+	var baseFee *big.Int
+	if header.BaseFee != nil {
+		baseFee = header.BaseFee
+	} else {
+		baseFee = new(big.Int).SetUint64(params.ZeroBaseFee)
+	}
 	intrinsicGas, err := types.IntrinsicGas(args.data(), nil, args.To == nil, b.ChainConfig().Rules(header.Number))
 	if err != nil {
 		return nil, 0, 0, err
 	}
-	msg, err := args.ToMessage(globalGasCap, fixedBaseFee, intrinsicGas)
+	msg, err := args.ToMessage(globalGasCap, baseFee, intrinsicGas)
```

Comment:
I think we need to change L1256 logic to calculate correct logic after hard fork.

Reply:
Okay, It is edited by following.
1. before hardfork, user should specify the gasPrice field.
2. after hardfork, user don't need to specify any field. However If specified, following priority will be applied, gasPrice, maxFeePerGas, baseFee in block header.

Reply:
Cool. Thank you for changing. I put more comments, please look into it. :)